modules/_labs/pagination/react/index.ts

```diff
@@ -1,5 +1,4 @@
-import Pagination from './lib/Pagination';
+import {Pagination} from './lib/Pagination';
 
 export default Pagination;
-export {Pagination};
 export * from './lib/Pagination';
```

Comment:
I'm fairly certain `export * from './lib/Pagination';` will cover `export {Pagination};` as it's a named export in `lib/Pagination`.

Reply:
Yeah. I've been pondering dropping default exports entirely in v5. We have to remap default exports so often I'm just not seeing the benefit.